libs/Objects/include/Objects/TransformationMonoid.h

```diff
@@ -16,96 +16,109 @@ class FiniteAutomaton;
 
 class TransformationMonoid {
   public:
-	struct Transition { // переход (индекс состояния - индекс состояния)
+	// переход (индекс состояния - индекс состояния)
+	struct Transition {
 		int first;
 		int second;
 		bool operator==(const Transition& a) const {
-			return this->first == a.first && this->second == a.second;
+			return first == a.first && second == a.second;
 		}
 		bool operator>(const Transition& a) const {
-			return this->first > a.first && this->second > a.second;
+			return first > a.first && second > a.second;
 		}
 		bool operator<(const Transition& a) const {
-			return this->first * 1000 + this->second <
-				   a.first * 1000 + a.second;
+			return first * 1000 + second < a.first * 1000 + a.second;
```

Comment:
учимся избавляться от магических констант
```suggestion
			return first < a.first || first == a.first && second < a.second;
```

Reply:
Пришел тимлид и все опошлил ...